src/go/k8s/pkg/client/acls/syncer.go

```diff
@@ -93,7 +93,7 @@ func (s *Syncer) sync(ctx context.Context, principal string, rules []redpandav1a
 func (s *Syncer) ListACLs(ctx context.Context, principal string) ([]redpandav1alpha2.ACLRule, error) {
 	describeResponse, err := s.listACLs(ctx, principal)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("listing ACLs: %w", err)
```

Comment:
NIT: I would drop that `listing ACLs` as it is duplicated in https://github.com/redpanda-data/redpanda-operator/pull/236/files#diff-8c966a8be080a7f0e6924dd74d9d34d294fe4c3191ed06a0ab144ff8cafba012R75